secretcache/cacheItem.go

```diff
@@ -129,6 +129,12 @@ func (ci *secretCacheItem) getVersion(versionStage string) (*cacheVersion, bool)
 	return secretCacheVersion, true
 }
 
+// refresh the cached object on demand
+func (ci *secretCacheItem) refreshNow(ctx context.Context) {
+	ci.refreshNeeded = true
```

Comment:
Do you think we need to add jitter similar to the [java implementation of refreshNow](https://github.com/aws/aws-secretsmanager-caching-java/blob/2d42f76af59568ec2151e000a27bd12f78a03c77/src/main/java/com/amazonaws/secretsmanager/caching/cache/SecretCacheObject.java#L214-L220)? I'm not saying it's necessary, just wanted to get your thoughts.

Reply:
I don't like the idea of protecting against possible misuse usecases, but since we already have the sleep on refresh if an exception happened, I've also added it here to be consistent